BotProject/Templates/CSharp/Schemas/update.cmd

```diff
@@ -0,0 +1,2 @@
+cd ..
+bf dialog:merge *.csproj -o schema/sdk.schema -v -b "R9"
```

Comment:
> "R9" [](start = 51, length = 5)

NO, this should be "" 
This switch is going away, and you should not be passing data to it, except an empty string.